config/locales/en.yml

```diff
@@ -951,6 +951,7 @@ en:
         view_products: Go To Products Page
         view_inventory: Go To Inventory Page
       product_headings:
+        distributor: Distributor
```

Comment:
It's good that it caught this. I'm wondering why it wasn't caught before. What is the difference between you config and this one?
https://github.com/openfoodfoundation/openfoodnetwork/blob/97985dbe8f118c2a80dae34cd727c6c380a01b6f/config/environments/test.rb#L39-L40

Reply:
From @dacook 's [comment](https://github.com/openfoodfoundation/openfoodnetwork/pull/11906#issuecomment-1856833265) above, I think that config line only catches exceptions on views and controllers.